source/deploy/mobile-faq.rst

```diff
@@ -372,3 +372,16 @@ In these scenarios, the ``login`` audit log won't have the ``deviceId``, and the
 If the app receives the ``deviceId`` later, the device will send the new ``deviceId`` to the server, generating an ``attachDeviceId`` audit log, and adding the ``deviceId`` to the session data in the database.
 
 Since the ``deviceId`` relates to the application, connections through the web browser, even on mobile, won't have a ``deviceID``.
+
+Where can I find mobile message notification logs?
+-------------------------------------------------------------
+Notification messages are logged to the ``notifications.log`` file.
+System admins can only access this ``notifications.log`` file by going to the ``File Location`` set in the ``Notification Settings``. The admins also have the privilege to set this file location and in case, no file location has been set, then the ``notifications.log`` file gets stored in the default working directory.
+Also, the ``notifications.log`` file exists only when it is enabled in Mattermost by setting ``NotificationSettings.EnableFile`` to ``true`` in the ``config.json`` file.
+You can learn more about logging `here <https://docs.mattermost.com/configure/environment-configuration-settings.html#logging>`__.
+
+The team members / users can access their notification logs based on their device platform.
+Android users can view the logs using ``logcat``.
+iOS users can acess the logs in the ``console app`` on their MacOS.
+However, the Android and iOS logs may differ from each other as different things are logged while receiving and processing requests based on the user platform.
```

Comment:
Let's remove this last line about the log file contents being different between platforms.

Reply:
Suggestions applied. Thanks.